src/main/java/net/unicon/lti/controller/lti/RegistrationController.java

```diff
@@ -87,8 +89,8 @@ public class RegistrationController {
      * @param model
      * @return
      */
-    @RequestMapping(value = {"", "/"}, method = RequestMethod.GET)
-    public String registration(@RequestParam("openid_configuration") String openidConfiguration, @RequestParam(name = LtiStrings.REGISTRATION_TOKEN, required = false) String registrationToken, HttpServletRequest req, Model model) {
+    @RequestMapping(value = {"", "/", "/{altDomain}"}, method = RequestMethod.GET)
```

Comment:
I realized that we might not need the path variable since you can extract the alt domain from the x-formatted-host. You don't have to make any changes, just a thought.

Reply:
I agree, we could use it but I still feel that from the user perspective and from the support one, having a specific registration url is a nice feature. And I don't trust 100% the header... I don't know, crazy network configurations, VPNs.. I don't know...  the altDomain is more reliable.